hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/BucketObjectDBInfo.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.recon.api.types;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.hadoop.hdds.client.DefaultReplicationConfig;
+import org.apache.hadoop.hdds.protocol.StorageType;
+import org.apache.hadoop.ozone.om.helpers.BucketEncryptionKeyInfo;
+import org.apache.hadoop.ozone.om.helpers.BucketLayout;
+
+/**
+ * Encapsulates the low level bucket info.
+ */
+public class BucketObjectDBInfo extends ObjectDBInfo {
+  /** volume name from om db. */
+  @JsonProperty("volumeName")
+  private String volumeName;
+
+  /** storage type for the object. */
+  @JsonProperty("storageType")
+  private StorageType storageType;
+
+  /** is versioning enabled for the object. */
+  @JsonProperty("versioning")
+  private boolean isVersioningEnabled;
+
+  /** used bytes for the object. */
+  @JsonProperty("usedBytes")
+  private String usedBytes;
+
+  /**
+   * Bucket encryption key info if encryption is enabled.
+   */
+  @JsonProperty("encryptionInfo")
+  private BucketEncryptionKeyInfo bekInfo;
+
+  /**
+   * Optional default replication for bucket.
+   */
+  @JsonProperty("replicationConfigInfo")
+  private DefaultReplicationConfig defaultReplicationConfig;
+
+  /** source volume. */
```

Comment:
Java doc comments, like this, don't provide any information as such. I would recommend to remove obvious comments like this everywhere.

Reply:
Thanks @hemantk-12 for review. Changes done. Pls re-review.